extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -43,11 +46,18 @@ public static function onMakeGlobalVariablesScript( &$vars ) {
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		$helper = new Helper();
 
+		/* Add assets for custom JS test mode */
 		if ( $helper->isContentReviewTestModeEnabled() || self::userCanEditJsPage() ) {
 			\Wikia::addAssetsToOutput( 'content_review_test_mode_js' );
 			\JSMessages::enqueuePackage( 'ContentReviewTestMode', \JSMessages::EXTERNAL );
 		}
 
+		/* Add Content Review Module assets for Monobook  */
+		if ( self::userCanEditJsPage() ) {
```

Comment:
It's covered by config `'skin' => [ 'monobook' ]`
https://github.com/Wikia/app/blob/CE-2538-review-status-module-monobook/extensions/wikia/AssetsManager/config.php#L2549-2549